demo/src/screens/componentScreens/TextFieldScreen/InputValidationsScreen.js

```diff
@@ -0,0 +1,82 @@
+import React, {Component} from 'react';
+import {StyleSheet, ScrollView} from 'react-native';
+import {View, Text, TextField, Spacings} from 'react-native-ui-lib';
+import {Navigation} from 'react-native-navigation';
+import {KeyboardAwareInsetsView} from 'react-native-keyboard-tracking-view';
+
+class InputValidationsScreen extends Component {
```

Comment:
Don't you want to add an `useTopErrors` example? Maybe a toggle button to switch between the top and the bottom error like in the input screen?

Reply:
done